packages/client/hmi-client/src/components/model-template/tera-model-template.vue

```diff
@@ -81,7 +81,7 @@ const cardWidth = computed(() => cardRef.value?.clientWidth ?? 0);
 const card = computed<ModelTemplateCard>(
 	() =>
 		props.model.metadata.templateCard ?? {
-			id: -1,
+			id: '',
```

Comment:
What is this `model.metadata.templateCard` business? Do we need it? If not it might be nice to clean up early before things get really gnarly with the junctions logic.

Reply:
I see ... let's have a quick chat when you have sometime. I think the name-spaces are getting a bit confusing with a lot of model and model-templates floating about. Let's see if we can make things a little easier to read.